src/libsystemd/sd-device/device-util.h

```diff
@@ -37,7 +37,7 @@
                 sd_device *_d = (device);                               \
                 int _level = (level), _error = (error);                 \
                                                                         \
-                if (_d && _unlikely_(log_get_max_level() >= _level))    \
+                if (_d && _unlikely_(log_get_max_level() >= LOG_PRI(_level))) \
```

Comment:
The change itself is ok. But I have a question, why this change is related to `SYNTHETIC_ERRNO()`?

Reply:
You're right, my commit message makes no sense. I got the level and errno fields crossed.

Reply:
Updated with the commit message rewrite.

Reply:
Thanks.